templates/repo/commit_statuses.tmpl

```diff
@@ -1,14 +1,27 @@
-{{if eq (len .Statuses) 1}}{{$status := index .Statuses 0}}{{if $status.TargetURL}}<a class="ui link commit-statuses-trigger vm" href="{{$status.TargetURL}}">{{template "repo/commit_status" .Status}}</a>{{end}}{{end}}
-<div class="ui commit-statuses-popup commit-statuses tippy-target">
-	<div class="ui relaxed list divided">
-		{{range .Statuses}}
-			<div class="ui item singular-status df">
-				{{template "repo/commit_status" .}}
-				<span class="ui ml-3 f1">{{.Context}} <span class="text grey">{{.Description}}</span></span>
-				{{if .TargetURL}}
-					<a class="ml-3" href="{{.TargetURL}}" target="_blank" rel="noopener noreferrer">{{$.root.locale.Tr "repo.pulls.status_checks_details"}}</a>
-				{{end}}
-			</div>
+{{if .Statuses}}
+	<span class="commit-statuses-trigger vm">
```

Comment:
@Ryuno-Ki has suggested add a `js-` prefix to classes that Javascript relies on in Forgejo chat.
It would need a change to the JS function initCommitStatuses in web_src/js/features/repo-commit.js

Reply:
Alternative: use a `data-` attribute.

```html
<span class="vm" data-tippy="commit-statuses">
```

Reply:
@fsologureng done in 6db8fb37400c14131235c27f7aaad36455600501